colossalai/inference/README.md

```diff
@@ -100,14 +101,9 @@ inference_config = InferenceConfig(
 engine = InferenceEngine(model, tokenizer, inference_config, verbose=True)
 
 # Step 4: try inference
-generation_config = transformers.GenerationConfig(
-            pad_token_id=tokenizer.pad_token_id,
-            max_new_tokens=512,
-        )
 prompts = ['Who is the best player in the history of NBA?']
-engine.add_request(prompts=prompts)
-response = engine.generate(generation_config)
-pprint(response)
+response = engine.generate(prompts)
+print(response)
```

Comment:
If you replace `pprint` with `print`, you should remove the line 82 `from pprint import pprint` as well.

Reply:
fixed